opds_import.py

```diff
@@ -691,46 +640,51 @@ def detail_for_elementtree_entry(cls, parser, entry_tag, feed_url=None):
         if identifier is None or not identifier.text:
             # This <entry> tag doesn't identify a book so we 
             # can't derive any information from it.
-            return None, None
+            return None, None, None
         identifier = identifier.text
 
-        # We will fill this dictionary with all the information
-        # we can find.
-        data = dict()
-
-        alternate_identifiers = []
-        for id_tag in parser._xpath(entry_tag, "dcterms:identifier"):
-            v = cls.extract_identifier(id_tag)
-            if v:
-                alternate_identifiers.append(v)
-        data['identifiers'] = alternate_identifiers
+        try:
+            # We will fill this dictionary with all the information
```

Comment:
And again here. Although actually writing those tests is likely to be a pain because of the etree code.